ghost/admin/app/validators/post.js

```diff
@@ -195,7 +195,7 @@ export default BaseValidator.create({
 
             // scheduled must be at least 2 mins in the future when first scheduling
```

Comment:
Can we update/remove this comment to avoid any confusion?

Reply:
```suggestion
            // scheduled must be in the future when first scheduling
```